build/repo.targets

```diff
@@ -16,6 +16,9 @@
     <JavacPath Condition="'$(OS)' != 'Windows_NT'">$(JAVA_HOME)/bin/javac</JavacPath>
 
     <HasJdk Condition="Exists('$(JavacPath)')">true</HasJdk>
+
+    <!-- Disable gradle daemon on CI since the CI seems to try to wait for the daemon to shut down, which it doesn't do :) -->
```

Comment:
We only disable when `CI` is `true` because we'd still like the Daemon to be in use on dev machines.

Reply:
By "the CI" do you mean VSTS? If so it sounds like the same thing we had a thread about this with the VSTS folks, and we have an issue filed with them that I'll link you to.

Reply:
No, TeamCity. VSTS works fine.

Reply:
It's either a TeamCity thing, a Universe thing (since it only occurs in TeamCity when building Universe/UpdateUniverse) or some combination of both.